src/benchmark.cc

```diff
@@ -177,6 +177,10 @@ State::State(IterationCount max_iters, const std::vector<int64_t>& ranges,
 #elif defined(__GNUC__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Winvalid-offsetof"
+#endif
+#if defined(__CUDACC__)
```

Comment:
Note: `nvcc` defines `__GNUC__` and emits `-Winvalid-offsetof` warning as well, so this has to be `#if` instead of `#elif`.